src/backends/sentry_backend_inproc.c

```diff
@@ -535,8 +535,8 @@ handle_ucontext(const sentry_ucontext_t *uctx)
         }
 
         if (should_handle) {
```

Comment:
I just noticed that we should `decref` (aka free) the event in the else branch. Probably this slipped by all the leak checkers since we are, well, crashing.

Reply:
Good catch!